mysite/templates/Includes/LoginBar.ss

```diff
@@ -1,41 +1,55 @@
+
+    <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
     <meta http-equiv="X-UA-Compatible" content="IE=9" />
     <section class="login-bar">
-        <div class="container">
-            <div class="row">
-                <div class="six columns">
-                    <h2>Welcome to the demo</h2>
-                </div>
-                <div class="six columns">
-                    <section class="login-form">
-                        <p>Press login to enter CMS</p>
-                        <form action="/Security/LoginForm" method="post" enctype="application/x-www-form-urlencoded" class="form-holder">
-                            <fieldset>
-                                <div class="form-item">
-                                    <label for="">Email address</label>
-                                    <input type="text" name="Email" placeholder="Email address" value="admin">
-                                </div>
-                                <div class="form-item">
-                                    <label for="">Password</label>
-                                    <input type="password" name="Password" placeholder="Password" value="password">
-                                </div>
-                            </fieldset>
-                            <button type="submit" name="button">Login</button>
-                            <span class="dropdown">
-                                <select name="Language">
-                                    <option value="admin" selected>EN – English</option>
-                                    <option value="german">DE – Deutsche</option>
-                                    <option value="spanish">ES – Español</option>
-                                    <option value="french">FR – Français</option>
-                                    <option value="tereo">MI – Māori</option>
-                                    <option value="russian">RU – русский</option>
-                                    <option value="arabic">AR – العربية</option>
-                                    <option value="chinese">ZH – 中文</option>
-                                    <option value="japanese">JA – 日本語</option>
-                                </select>
-                            </span>
-                        </form>
-                    </section>
-                </div>
-            </div>
+        <a href="http://www.silverstripe.org/">
+            <svg class="logo-icon" viewBox="0 0 99.31 100">
```

Comment:
Are there examples of other places where we have directly embedded SVGs? http://caniuse.com/#search=svg says IE8 doesn't have native SVG support